src/components/Title.tsx

```diff
@@ -1,5 +1,20 @@
 import React from 'react';
+import { BaseComponentProps } from '@/types';
 
-const Title: React.FC = () => <h1>Hello TypeScript!</h1>;
+export interface TitleProps extends BaseComponentProps {
+  testId?: string;
```

Comment:
We can remove this `testId` logic. See the other comment for more info.

Reply:
Done